src/envoy/http/jwt_auth/jwt_authenticator.cc

```diff
@@ -60,14 +60,14 @@ void JwtAuthenticator::Verify(HeaderMap& headers,
   callback_ = callback;
 
   // Sanitize the JWT verification result in the HTTP headers
-  for (const auto& rule : store_.config().rules()) {
-    if (!rule.forward_payload_header().empty()) {
-      ENVOY_LOG(debug, "Sanitize JWT authentication output header {}",
-                rule.forward_payload_header());
-      const LowerCaseString key(rule.forward_payload_header());
-      headers.remove(key);
-    }
-  }
+  // for (const auto& rule : store_.config().rules()) {
+  //   if (!rule.forward_payload_header().empty()) {
+  //     ENVOY_LOG(debug, "Sanitize JWT authentication output header {}",
+  //               rule.forward_payload_header());
+  //     const LowerCaseString key(rule.forward_payload_header());
+  //     headers.remove(key);
+  //   }
+  // }
```

Comment:
No dead code please.   If code is not needed, please remove it.

Reply:
Done